packages/astro/src/core/errors/dev/vite.ts

```diff
@@ -187,3 +191,18 @@ export async function getViteErrorPayload(err: ErrorWithMetadata): Promise<Astro
 		return errorName.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase();
 	}
 }
+
+function findSuggestedImportPaths(failedImportString: string, currentFilePath: string): string[] {
+	const suggestedPaths: string[] = [];
+	const prefixes = ["../../", "../", "./", "/", "~/"]
```

Comment:
I don't think `~/` is always supported, assuming that's a custom alias?

Reply:
Removed e3388d8082d4823a3fa638cd75a099a0d098af26 👍